pulp_rpm/tests/functional/content_handler/test_config_repo.py

```diff
@@ -60,3 +83,15 @@ def test_config_repo_auto_distribute(
     # Delete publication and check that 404 is now returned
     rpm_publication_api.delete(pub.pulp_href)
     assert requests.get(f"{dist.base_url}config.repo").status_code == 404
+
+
+@pytest.mark.parallel
+@pytest.mark.skipif(
+    not (settings.CONTENT_ORIGIN == ""),
```

Comment:
It was left as `None` in the end. [[0]](https://github.com/pulp/pulpcore/commit/115e6c58b6a67781e61e29256906997516adcec0#diff-4e1146f075426039ac8e1350e88fe5736a3e50872a42d101bfc8bc1b7f483574)

We should change this so the test is skipped correctly